### DISCUSSION TITLE
Make Outline tree expandable

### DISCUSSION THREAD

**Discussion by Prosolve** [IgniteUI/app-builder#25] [Ideas]
Would be handy to have the Outline tree expandable rather than just a scrollbar as it can be tricky to see the heirachy sometimes.

First post here on GitHub, wasn't sure if this or Discord was the best place.

**Comment by zdrawku** [comment]
Thanks, Prosolve, both places are fine, here other people can benefit in the future as the history is tracked better.

The outline panel can be expanded by dragging the splitter handle. Going further, for the time being we found valuable only having a `Collapse all` button as `Expand all` might lead to overcrowding the whole Outline panel. 

![image](https://github.com/IgniteUI/app-builder/assets/11193764/4e197c2c-b819-4044-bdf1-0aaaf25fa8f6)

**Reply by Prosolve** [reply]
Ah I meant the width of the panel

**Reply by zdrawku** [reply]
I see now, thanks for the clarification. 

This is a feature request. Sharing a brief acceptance criteria:
- Provide the ability **to change** the left-side panel **width**. Define a **min-width** value.
- Consider exposing **overlay** behavior for the left-side panel.

**Reply by tommyjr8989** [reply]
> Thanks, Prosolve, both places are fine, here other people can benefit in the future as the history is tracked better.
> 
> The outline panel can be expanded by dragging the splitter handle. Going further, for the time being we found valuable only having a `Collapse all` button as `Expand all` might lead to overcrowding the whole Outline panel. 
> 
> ![image](https://github.com/IgniteUI/app-builder/assets/11193764/4e197c2c-b819-4044-bdf1-0aaaf25fa8f6)
>

**Comment by zdrawku** [comment]
Marking the discussion as feature request
